fe/fe-core/src/main/java/org/apache/doris/common/util/PropertyAnalyzer.java

```diff
@@ -213,7 +222,7 @@ public static DataProperty analyzeDataProperty(Map<String, String> properties, D
         }
 
         Preconditions.checkNotNull(storageMedium);
-        return new DataProperty(storageMedium, cooldownTimeStamp, remoteStorageResourceName, remoteCooldownTimeStamp);
+        return new DataProperty(storageMedium, cooldownTimeStamp, remoteStorageResourceName, remoteCooldownTimeStamp, storagePolicy);
```

Comment:
If MigrationHandler is on BE, remoteStorageResourceName and remoteCooldownTimeStamp are not useful, just remove them from DataProperty.